docs/install/installEverest.md

```diff
@@ -72,14 +72,52 @@ To install and provision Percona Everest to Kubernetes:
             ??? example "Expected output"
                 ```
                 NAME      TYPE           CLUSTER-IP      EXTERNAL-IP     PORT(S)          AGE
-                everest   LoadBalancer   10.43.172.194   127.0.0.1       8080:8080/TCP    10s
+                everest   LoadBalancer   10.43.172.194   34.175.201.246       8080:8080/TCP    10s
                 ```
 
+
+    === "Node Port"
+        A NodePort is a service that makes a specific port accessible on all nodes within the cluster. It enables external traffic to reach services running within the Kubernetes cluster by assigning a static port to each node's IP address.
+
+        1. Run the following command to change the Everest service type to `NodePort`:
+
+            ```sh
+            kubectl patch svc/everest -n everest-system -p '{"spec": {"type": "NodePort"}}
+            ```
+
+        2. Assign the external port to the Everest service, which is `32349` in this case.
+
+            ```sh
+            kubectl get svc/everest -n everest-system
+            NAME      TYPE       CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
+            everest   NodePort   10.43.139.191   <none>        8080:32349/TCP   28m
+            ```
+
+        3. Retrieve the external IP addresses for the kubernetes cluster nodes.
+
+            ```sh
+            kubectl get nodes -o wide
+            NAME                   STATUS   ROLES    AGE   VERSION             
+            INTERNAL-IPEXTERNAL-IP  OS-IMAGE                        KERNEL-VERSION   
+            CONTAINER-RUNTIME
+            gke-everest-test-default-pool-8bbed860-65gx   Ready    <none>   3m35s   
+            v1.30.3-gke.1969001   10.204.15.199   34.175.155.135   Container- 
+            Optimized OS from Google   6.1.100+         containerd://1.7.19
+            gke-everest-test-default-pool-8bbed860-pqzb   Ready    <none>   3m35s   
+            v1.30.3-gke.1969001   10.204.15.200   34.175.120.50    Container- 
+            Optimized OS from Google   6.1.100+         containerd://1.7.19
+            gke-everest-test-default-pool-8bbed860-s0hg   Ready    <none>   3m35s   
+            v1.30.3-gke.1969001   10.204.15.201   34.175.201.246   Container- 
+            Optimized OS from Google   6.1.100+         containerd://1.7.19
+            ```
+        
+        4. To launch the Percona Everest UI and create your first database cluster, go to the IP address/port found in steps 2 and 3. In this example, the external IP address is [http://34.175.155.135:32349](http://34.175.155.135:32349).
```

Comment:
We should state that any node IP will work but we'll use the first one as the example.

Reply:
Fixed